.circleci/config.yml

```diff
@@ -90,6 +90,8 @@ workflows:
       - deploy-dev:
           requires:
             - build-test-dev
+          context:
+            - sig-go-release
```

Comment:
Should we split this into a separate PR?  Sure, it's trivial, but I'd like to squash the other commits in this PR, and this one should be kept separate (as it's an entirely different/unrelated thought/change).

Reply:
https://github.com/pantheon-systems/fusedav/pull/518